tests/getDeploy.test.ts

```
@@ -94,9 +89,13 @@ describe('#getDeploy', () => {
         initialManager:
           '0x7AcaF5360474b8E40f619770c7e8803cf3ED1053' as `0x${string}`,
       },
-      paymentProcessor: {
-        epochLength: BigInt(604800), // 1 week in seconds
-      },
+      optionalModules: [
+        {
+          name: 'PaymentProcessor',
+          version: '1',
+          params: { epochLength: BigInt(604800) },
```

this should throw an error